src/components/InstructorRevenueCard/index.js

```diff
@@ -0,0 +1,123 @@
+
+                        <div className='mb4'>
+                          <RevenuePeriod
+                            title={<Text message='instructorRevenue.currentMonth.title' />}
+                            revenue={currentMonthRevenue.revenue}
+                            subscriberMinutes={currentMonthRevenue.minutes_watched}
+                          />
+                        </div>
+
+                        <RevenuePeriod
+                          title={<Text 
+                            message='instructorRevenue.previousMonths.title' 
+                            values={[currentTotalRevenue.monthCount]} 
+                          />}
+                          revenue={currentTotalRevenue.revenue}
+                          subscriberMinutes={currentTotalRevenue.minutes_watched}
+                        />
+
+                        <div className='mt4'>
+                          <Button 
+                            onClick={handleOpenToggleClick}
+                            size='extra-small'
+                          >
+                            <Text message='instructorRevenue.action' />
+                          </Button>
+                        </div>
+
+                      </div>
+                  }
+
+                  {isOpen
+                    ? <div className='pa4 w-100'>
+
+                        <Heading level='4'>
+                          <Text message='instructorRevenue.revenue' />
+                        </Heading>
+                        <LineChart
+                          xAxis={currentMonthDates}
+                          yAxis={[
+                            {
+                              color: revenueColor,
+                              points: currentRevenuePoints,
+                            },
+                          ]}
+                          type='detail'
+                        />
+
+                        <div className='mt4'>
+                          <Heading level='4'>
+                            <Text message='instructorRevenue.minutesWatched' />
+                          </Heading>
+                        </div>
+                        <LineChart
+                          xAxis={currentMonthDates}
+                          yAxis={[
+                            {
+                              color: minutesColor,
+                              points: currentMinutesPoints,
+                            },
+                          ]}
+                          type='detail'
+                        />
+
+                      </div>
+                    : <LineChart
+                      xAxis={currentMonthDates}
+                      yAxis={[
+                        {
+                          color: revenueColor,
+                          points: currentRevenuePoints,
+                        },
+                      ]}
+                      className='bg-white-secondary br2 br--bottom'
+                    />
+                  }
```

Comment:
See three different `<LineChart` instances above; this is the abstraction I made over the ChartJS which requires a lot more config. So now you just need to pass the axis's and `type` (defaults to `simple` but can be `detail`).
Could be a good candidate for promotion to egghead-ui, per the work @tayiorbeii has been doing with chartjs.